assets/scss/variables/_colors.scss

```diff
@@ -1,19 +1,37 @@
-$colors: (
-    white: #fff,
-    black: #010101,
+// Export variables from variable2css figma plugin
+// https://www.figma.com/community/plugin/1261234393153346915/variables2css
 
-    grey-50: #F6F6F6,
-    grey-100: #F0F0F0,
-    grey-200: #E3E3E3,
-    grey-300: #D1D1D1,
-    grey-400: #9F9F9F,
-    grey-500: #757575,
-    grey-600: #616161,
-    grey-700: #4F4F4F,
-    grey-800: #313131,
-    grey-900: #1D1D1D,
+/* Primitives */
+$colors-black-0: rgb(1 1 1 / 0%);
+$colors-black-5: rgb(1 1 1 / 5%);
+$colors-black-10: rgb(1 1 1 / 10%);
+$colors-black-15: rgb(1 1 1 / 15%);
+$colors-black-20: rgb(1 1 1 / 20%);
+$colors-black-30: rgb(1 1 1 / 30%);
+$colors-black-50: rgb(1 1 1 / 50%);
+$colors-black-80: rgb(1 1 1 / 80%);
+$colors-black-100: rgb(1 1 1);
+$colors-gray-100: rgb(240 240 240);
+$colors-gray-200: rgb(227 227 227);
+$colors-gray-300: rgb(209 209 209);
+$colors-gray-400: rgb(159 159 159);
+$colors-gray-50: rgb(246 246 246);
+$colors-gray-500: rgb(117 117 117);
+$colors-gray-600: rgb(97 97 97);
+$colors-gray-700: rgb(79 79 79);
+$colors-gray-800: rgb(49 49 49);
+$colors-white-0: rgb(255 255 255 / 0%);
+$colors-white-10: rgb(255 255 255 / 10%);
+$colors-white-100: rgb(255 255 255);
+$colors-white-15: rgb(255 255 255 / 15%);
+$colors-white-20: rgb(255 255 255 / 20%);
+$colors-white-30: rgb(255 255 255 / 30%);
+$colors-white-5: rgb(255 255 255 / 5%);
+$colors-white-50: rgb(255 255 255 / 50%);
+$colors-white-80: rgb(255 255 255 / 80%);
+$colors-error: rgb(244, 67, 54);
+$colors-warning: rgb(245, 133, 97);
+$colors-success: rgb(34, 187, 150);
```

Comment:
IMO we should delete all the default variables

Reply:
We use these colors in some components, so I think we need to keep them to avoid errors. They will be deleted after the variables have been exported from Figma